gap/oper.gi

```diff
@@ -96,8 +96,8 @@ function(D, u)
   if u > DigraphNrVertices(D) then
     return D;
   fi;
-  Remove(D!.OutNeighbours, u);
   RemoveDigraphVertexLabel(D, u);
+  Remove(D!.OutNeighbours, u);
   for v in DigraphVertices(D) do
```

Comment:
Is there any reason these lines are swapped? If not, please undo it.

Reply:
This was the thing I mentioned in Slack, where the function wasn't behaving as expected until I swapped them. I'm not sure about the underlying reason for the problem, though.